kube-core/src/params.rs

```diff
@@ -140,6 +140,33 @@ impl ListParams {
     }
 }
 
+/// The validation directive to use for `fieldValidation` when using server-side apply.
+#[derive(Clone, Debug)]
+pub enum ValidationDirective {
+    /// This will fail the request with a BadRequest error if any unknown fields would be dropped from the
+    /// object, or if any duplicate fields are present. The error returned from the server will contain
+    /// all unknown and duplicate fields encountered.
```

Comment:
The first line of these will show up in rustdoc so it would be good if the docs here had the following format:

```
/// Introductory sentence
///
/// full explaination that shows up when clicking into the element
Strict,
```

Reply:
Thanks! Cleaned up a few more doc blocks, hopefully that all made sense